docs/chapters/licensing/index.rst

```diff
@@ -0,0 +1,25 @@
+Licensing
+=========
+
+Each source code file needs to mention what license applies and who owns
+the copyright in the file header. The purpose of this is to make it
+absolutely clear, for both humans and tooling, which license it is that
+applies to that particular file even when the file is moved to another
+project. In order to achieve this, it has been decided that the header
+should include:
```

Comment:
I agree with @GordanM if we remove the part of having discussed it we do not really mitigate another discussion starting on the subject. However, I guess you could argue the same for all the docs... Need to think this through.